### DISCUSSION TITLE
Generate HTML DOM from String, similiar to dangerouslySetInnerHTML in React

### DISCUSSION THREAD

**Discussion by thanhdatvo** [solidjs/solid#1459] [Q&A]
Hi,
In react we could generate HTML DOM from String using dangerouslySetInnerHTML
Is there any way to do that in SolidJs

Thank you,

**Comment by maxmilton** [comment]
You probably want the `innerHTML` attribute. Check out the docs: <https://www.solidjs.com/docs/latest/api#innerhtmltextcontent>.

Alternatively, you can [get a ref to an element](https://www.solidjs.com/docs/latest/api#ref) and use plain JS.

**Reply by thanhdatvo** [reply]
exactly what's I'm looking for!
Thank you a lot!
